lib/elixir/lib/float.ex

```diff
@@ -8,6 +8,8 @@ defmodule Float do
   import Bitwise
 
   @power_of_2_to_52 4503599627370496
+  @precision_valid_range 0..15
```

Comment:
`_valid_` part feels redundant, I believe there are many cases where we simply omit it.